contracts/ThalesRoyale/ThalesRoyale.sol

```diff
@@ -497,6 +571,11 @@ contract ThalesRoyale is Initializable, ProxyOwned, PausableUpgradeable, ProxyRe
         emit NewPriceFeed(_priceFeed);
     }
 
+    function setThalesRoyalePassport(IThalesRoyalePassport _thalesRoyalePassport) public onlyOwner {
+        thalesRoyalePassport = _thalesRoyalePassport;
```

Comment:
check zero addresses (i learn that in a audit), and add it for the rest if needed 🙏

Reply:
i saw you did it in a passport contract